### ISSUE TITLE
[amp-story-interactive] Allow for not displaying results

### ISSUE THREAD

**Issue by cpauwels** [ampproject/amphtml#32544]
## Describe the new feature or change to an existing feature you'd like to see

There are some cases for polls or quizes where the story publisher might not want to show the results. A simple case would be one of polling for customer satisfaction. If customers are not satisfied, the publisher might not want to show the results or make them public. 

I think simply removing the response percentage text should do the trick, and having the visualisation similar to just one person having responded (100% for the selected option, 0% for all others)

**Comment by newmuis** [comment]
@mszylkowski I know if no backend is specified, we (obviously) don't aggregate the results; is it possible to trigger that state while still sending data to the backend?

**Comment by mszylkowski** [comment]
On our v1 launch we designed components to always show the responses but I think this is a completely valid use-case.

Currently the way I see this working in this way would be to have the backend return an invalid JSON (but still record the vote), so the percentages don't show but they get recorded in the backend. This requires some development from the backend owner but I think is the more robust/secure solution (and the format can gracefully catch this error, but it should currently work as you detailed).

We can also introduce an attribute to hide the results in the component, but anyone would be able to know the customer responses with the right HTTP request. It should be relatively easy to make this change and we can add it to our roadmap.

**Comment by newmuis** [comment]
Good point about changing the endpoint's response.  Sounds reasonable to launch an attribute to disable the result state and provide documentation on the shape of the JSON response in that case.

/cc @raovs @hongcatlover to ensure this sounds reasonable to them

**Comment by raovs** [comment]
LGTM from my side
